dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -109,4 +109,8 @@ plugin.sequence.java.util.Collection = \
 # Enable a test authority control on dc.language.iso field
 choices.plugin.dc.language.iso = common_iso_languages
 choices.presentation.dc.language.iso = select
-authority.controlled.dc.language.iso = true
\ No newline at end of file
+authority.controlled.dc.language.iso = true
+
+# Enable a test authority control on dc.type field
+choices.plugin.dc.type = common_types
+choices.presentation.dc.type = select
```

Comment:
These additions should go to `dspace.cfg`

Reply:
Maybe shouldn't still exists because now the authority related to valuepairs is auto discover. I have to check if this it used in some test.